js/style/style.js

```diff
@@ -63,7 +63,7 @@ function Style(stylesheet, animationLoop) {
 
         if (stylesheet.sprite) {
             this.sprite = new ImageSprite(stylesheet.sprite);
-            this.sprite.on('load', this.fire.bind(this, 'change'));
+            this.sprite.on('load', this.fire.bind(this, ['change', 'sprites.quiesce']));
```

Comment:
The goal of #1715 is to fire a `quiesce`event on the `Map` object when both the sprites and the tiles have "quiesced" (there are no outstanding network requests for any sprites or tiles).

Here, you're correctly firing an event when the sprites "quiesce" but we want the event to fire when both sprites and tiles have "quiesced", as indicated by the `this._loaded()` method (which should really be renamed to `this._isQuiesced()` now) 

```js
this.sprite.on('load', this.fire.bind(this, 'change'));
this.sprite.on('load', function() {
    if (this.loaded()) {
        this.fire('quiesce');
    }
});
```